docs/quick-start/new-project/README.md

```diff
@@ -0,0 +1,268 @@
+
+Run `npm install`. 
+
+Mira and the AWS CDK are now installed.
+
+## Configure Mira
+Once you have Mira installed, you can invoke it via the `npx` command. For example, to view the built-in documentation run `npx mira docs`. This presents you with a local URL that you use in your browser.
+
+To configure Mira you need to create a default configuration file. You need your AWS account number and chosen deployment region information to hand. Run `npx mira init` to start the configuration wizard. Enter appropriate values when prompted to replace the sample values shown below:
+```
+This utility will walk you through creating a default.json file.
+ 
+Please consult the Mira documentation for definitive documentation
+on these fields and exactly what they do.
+ 
+Press ^C at any time to quit.
+ 
+? Application Name? s3-website
+? Application Prefix? nf
+? CI/CD Environment AWS Account ID? 1234567890
+? CI/CD Environment AWS Account Region? eu-west-1
+? CI/CD Environment local AWS CLI configuration profile name? default
+Successfully created config/default.json
+```
+This will create a Mira configuration file named `./config/default.json` containing the minimum needed to deploy an application. Refer to the [configuration section](/config/README.md) for a full description of each field.
+
+## Build your website
+This example uses a very basic `index.html` page to illustrate the deployment, but this could be the build output of a React or indeed any other application. In the root of the project, create a new folder called `web-app`. Inside, create an `index.html` file with content of your choosing, for example:
+
+`cat web-app/index.html`
+```html
+<!DOCTYPE html>
+<html>
+    <head>
+        <title>S3 Website</title>
+    </head>
+    <body>
+        <h1>S3 Website</h1>
+        <p>Welcome to my S3 powered website.</p>
+    </body>
+</html>
+```
+We can also include a custom error page for 404 and the like, for example:
+`cat web-app/error.html`
+```html
+<!DOCTYPE html>
+<html>
+    <head>
+        <title>S3 Website</title>
+    </head>
+    <body>
+        <h1>ERROR</h1>
+    </body>
+</html>
+```
+## Set up TypeScript
+Create a folder called `infra` with a sub-directory called `src`. This contains the code required to define and deploy your infrastructure. Mira uses TypeScript to  integrate with the AWS CDK and you need to complete a small number of steps to set that up. In the `package.json` file, include TypeScript and the type definitions for Node to the development dependencies as follows: 
+```json
+ "devDependencies": {
+    "@types/node": "^10.17.24",
+    "typescript": "^3.9.5"
+  }
+```
+While editing `package.json`, also add the following helpful scripts which you will need later: 
+```json
+ "scripts": {
+    "build": "tsc -p infra/",
+    "dev": "tsc -p infra/ --watch",
+    "deploy": "npx mira deploy s3-website --file=infra/src/index.js",
+    "undeploy": "npx mira undeploy s3-website --file=infra/src/index.js"
+  },
+```
```

Comment:
Wondering if we could automate this as similar to: nearform/mira-archive#429

Reply:
Yes, I would add it to that ticket